### ISSUE TITLE
Details of PolynomialLRDecay

### ISSUE THREAD

**Issue by kleinzcy** [frazerlin/fcanet#1]
Thanks for your nice work!!!
 In the paper, you use the polynomial learning decay for 30 epochs, but you have not mention the end learning rate of PolynomialLRDecay. Can you offer the details? If possible, I hope you can open your training code to fill the blank. (In Interactive segmentation, so many works only public the test code.)
My email is zcy_ai@whu.edu.cn.
Thanks.
